lib/utils/index.js

```diff
@@ -533,6 +533,29 @@ class Utils {
   static shouldReplaceStack(err) {
     return !alwaysDisplayError(err);
   }
+
+  static findTSConfigFile() {
+    const projectTsFileLocation1 = path.join(process.cwd(), 'nightwatch', 'tsconfig.json');
+    const projectTsFileLocation2 = path.join(process.cwd(), 'tsconfig.nightwatch.json');
+    if (Utils.fileExistsSync(projectTsFileLocation1)) {
+      return projectTsFileLocation1;
+    } else if (Utils.fileExistsSync(projectTsFileLocation2)) {
+      return projectTsFileLocation2;
+    }
+
+    return '';
+  }
```

Comment:
```suggestion
  static findTSConfigFile() {
    const allowedFileLocation = [path.join('nightwatch', 'tsconfig.json'), 'tsconfig.nightwatch.json'];
    const filesPresent = allowedFileLocation
      .map((fileLocation) => path.join(process.cwd(), fileLocation))
      .filter((filePath) => Utils.fileExistsSync(filePath));
  
    return filesPresent[0] || '';
  }
```

Reply:
I am in favour of the existing implementation which is a bit simpler and easier to understand. Besides, `.filter()` will search the whole array even if the first file is used. We should use `Array.some()` instead.